ghost/webmentions/lib/MentionsAPI.js

```diff
@@ -160,8 +162,9 @@ module.exports = class MentionsAPI {
                 sourceFavicon: metadata.favicon,
                 sourceFeaturedImage: metadata.image
             });
+            const event = MentionCreatedEvent.create({data: mention, timestamp: new Date()});
```

Comment:
Can we update this to use the pattern we use in Tiers?

First we add the Event to the Tier object like this: https://github.com/TryGhost/Ghost/blob/main/ghost/tiers/lib/Tier.js#L282-L284

And then in the TierRepository, we dispatch all necessary events: https://github.com/TryGhost/Ghost/blob/main/ghost/core/core/server/services/tiers/TierRepository.js#L112-L114